iset.mm

```diff
@@ -155588,6 +155642,514 @@ the additive structure must be abelian (see ~ ringcom ), care must be
+    lss1d.s $e |- S = ( LSubSp ` W ) $.
+    $( One-dimensional subspace (or zero-dimensional if ` X ` is the zero
+       vector).  (Contributed by NM, 14-Jan-2014.)  (Proof shortened by Mario
+       Carneiro, 19-Jun-2014.) $)
+    lss1d $p |- ( ( W e. LMod /\ X e. V )
+        -> { v | E. k e. K v = ( k .x. X ) } e. S ) $=
+      ( wcel wa cfv co wceq cvv vx vj va vb vy vz clmod cplusg cv wrex cab csca
+      a1i cbs eqidd cvsca clss lmodvscl 3expa an32s eleq1a syl rexlimdva abssdv
+      wi wex c0g eqid lmod0cl elex2 adantr nfv nfre1 nfsab nfex vscaslid slotex
+      vex eqeltrid simpr ovexg mp3an2i elabrexg sylan2 expcom mpd eqeq1 rexbidv
+      exlimd w3a oveq1 eqeq2d cbvrexvw bitri anbi12i reeanv bitr4i cmulr simpll
+      simprr simprll lmodmcl syl3anc simprlr lmodacl simplr lmodvsdir lmodvsass
+      elab syl13anc oveq1d eqtr2d rspceeqv oveq2 oveq12 sylan eqeq1d syl5ibrcom
+      syl2anc expr com23 rexlimdvva biimtrid expcomd com24 3imp2 elabg ad2antrr
+      wb plusgslid mpbird simpl islssmd ) HUGOZIGOZPZUAFHUHQZBCAUIZDUIZICRZSZDF
+      UJZAUKZUBEGHUGUCUDEHULQSYPKUMFEUNQSYPMUMGHUNQSYPJUMYPYQUOCHUPQZSYPLUMBHUQ
+      QSYPNUMYPUUBAGYPUUAYRGOZDFYPYSFOZPYTGOZUUAUUEVEYNUUFYOUUGYNUUFYOUUGYSCEFG
+      HIJKLMURUSUTYTGYRVAVBVCVDYPUUFDVFZUBUIUUCOZUBVFZYNUUHYOYNEVGQZFOUUHEFHUUK
+      KMUUKVHVIDUUKFVJVBVKYPUUFUUJDYPDVLUUIDUBUUBDAUBUUADFVMVNVOUUFYPUUJUUFYPPY
+      TUUCOZUUJYPUUFYTTOZUULYSTOYPCTOZYOUUMDVRYNUUNYOYNCUUDTLHUPUGVPVQVSZVKYNYO
+      VTYSICTTGWAWBDAFYTTWCWDUBYTUUCVJVBWEWIWFYPUAUIZFOZUCUIZUUCOZUDUIZUUCOZWJZ
+      PUUPUURCRZUUTYQRZUUCOZUVDYTSZDFUJZYPUUQUUSUVAUVGYPUVAUUSUUQUVGYPUUSUVAUUQ
+      UVGVEZUUSUVAPZUURUEUIZICRZSZUUTUFUIZICRZSZPZUFFUJUEFUJZYPUVHUVIUVLUEFUJZU
+      VOUFFUJZPUVQUUSUVRUVAUVSUUSUURYTSZDFUJZUVRUUBUWAAUURUCVRZYRUURSUUAUVTDFYR
+      UURYTWGWHXIUVTUVLDUEFYSUVJSYTUVKUURYSUVJICWKWLWMWNUVAUUTYTSZDFUJZUVSUUBUW
+      DAUUTUDVRZYRUUTSUUAUWCDFYRUUTYTWGWHXIUWCUVODUFFYSUVMSYTUVNUUTYSUVMICWKWLW
+      MWNWOUVLUVOUEUFFFWPWQYPUVPUVHUEUFFFYPUVJFOZUVMFOZPZPUUQUVPUVGYPUWHUUQUVPU
+      VGVEYPUWHUUQPZPZUVGUVPUUPUVKCRZUVNYQRZYTSZDFUJZUWJUUPUVJEWRQZRZUVMEUHQZRZ
+      FOZUWLUWRICRZSUWNUWJYNUWPFOZUWGUWSYNYOUWIWSZUWJYNUUQUWFUXAUXBYPUWHUUQWTZY
+      PUWFUWGUUQXAZUWOEFHUUPUVJKMUWOVHZXBXCZYPUWFUWGUUQXDZUWQEFHUWPUVMKMUWQVHZX
+      EXCUWJUWTUWPICRZUVNYQRZUWLUWJYNUXAUWGYOUWTUXJSUXBUXFUXGYNYOUWIXFZYQUWQUWP
+      UVMCEFGHIJYQVHKLMUXHXGXJUWJUXIUWKUVNYQUWJYNUUQUWFYOUXIUWKSUXBUXCUXDUXKUUP
+      UVJCUWOEFGHIJKLMUXEXHXJXKXLDUWRFYTUWTUWLYSUWRICWKXMXSUVPUVFUWMDFUVPUVDUWL
+      YTUVLUVCUWKSUVOUVDUWLSUURUVKUUPCXNUVCUWKUUTUVNYQXOXPXQWHXRXTYAYBYCYDYEYFY
+      NUVEUVGYIZYOUVBYNUVDTOZUXLYNUVCTOZYQTOUUTTOZUXMUUPTOYNUUNUURTOZUXNUAVRUUO
+      UXPYNUWBUMUUPUURCTTTWAWBHUHUGYJVQUXOYNUWEUMUVCUUTYQTTTWAXCUUBUVGAUVDTYRUV
+      DSUUAUVFDFYRUVDYTWGWHYGVBYHYKYNYOYLYM $.
+  $}
+
+  ${
+    $d a b w x y A $.  $d a b x y S $.  $d a b w x y W $.
+    lssintcl.s $e |- S = ( LSubSp ` W ) $.
+    $( The intersection of a nonempty set of subspaces is a subspace.
```

Comment:
nonempty -> inhabited?

Reply:
inabited => inhabited

Reply:
Do I get partial credit for spelling it correctly in the git commit message?

Anyway, fixed.

Reply:
That's the french pronunciation, it works for me :)